specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json

```diff
@@ -511,12 +567,114 @@
               "x-ms-version": {
                 "x-ms-client-name": "Version",
                 "type": "string",
-                "description": "Indicates the version of the Queue service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
+                "description": "Indicates the version of the Table service used to execute the request. This header is returned for requests made against version 2009-09-19 and above."
               },
               "Date": {
                 "type": "string",
                 "format": "date-time-rfc1123",
-                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated"
+                "description": "UTC date/time value generated by the service that indicates the time at which the response was initiated."
+              },
+              "ETag": {
+                "type": "string",
+                "description": "UTC date/time value generated by the service that indicates the time at which the entity was last updated."
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/TableServiceError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "table"
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "operationId": "Table_MergeEntity",
+        "x-ms-examples": {
+          "TableMergeEntity": {
+            "$ref": "./examples/TableMergeEntity.json"
```

Comment:
Do we need to add this file as a part of this PR?

Reply:
Good catch - added. Also updated the Delete example with the added required parameter.